### PR TITLE
LPS-170578 | Create automated test for ClayAlertToast

### PR DESCRIPTION
**Pull request by mateussandes23**
**Description:**
- Assert that alert message remain when mouse hover over it

**Test Plan:**
- Given Clay Sample Portlet
- When a toast alert appears
- And user mouses over the alert
- Then the alert does not disappear when hovered over

**JIRA ticket:**
https://issues.liferay.com/browse/LPS-170578

cc/ @john-co @mateusralv

### CODE REVIEW

**Comment by mateusralv** [comment]
ci:test:frontend

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+				locator1 = "Button#ANY");
+		}
+
+		task ("And user mouses over the alert") {
+			MouseOver(locator1 = "Message#SUCCESS");
+		}
+
+		task ("Then the alert does not disappear when hovered ove") {
+			AssertElementPresent(locator1 = "Message#SUCCESS");
+		}
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase:54]: This doesn't make sure that the alert doesn't disappear when hovered over. It just checks immediately if the element is present. I need you to think how can you make the test make sure that the message hasn't disappeared.

**Review by mateusralv — CHANGES_REQUESTED** [review]
Add just a few things, the test looks good to me 😄

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+			Click(
+				key_text = "Success Submit",
+				locator1 = "Button#ANY");
+		}
+
+		task ("And user mouses over the alert") {
+			MouseOver(locator1 = "Message#SUCCESS");
+		}
+
+		task ("Then the alert does not disappear when hovered ove") {
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase:52]: typo: `over`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONLayout.deletePublicLayout(
+				groupName = "Guest",
+				layoutName = "Clay Page");
+		}
+	}
+
+	@description = "LPS-170578 - Assert that message remain after mouse is away from it."
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase:33]: please correct the description: to match the test

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+		property portal.acceptance = "true";
+
+		task ("When a toast alert appear") {
+			WaitForElementPresent(
+				key_text = "Success Submit",
+				locator1 = "Button#ANY");
+
+			Click(
+				key_text = "Success Submit",
+				locator1 = "Button#ANY");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase:45]: Like we discussed before, please use fail button/message on Clay Sample portlet

Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase:45]: Success message will not autoclose.

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:frontend-clay

**Comment by john-co** [comment]
ci:test:frontend:nocompile

**Comment by john-co** [comment]
testing again in https://github.com/liferay-frontend/liferay-portal/pull/2929 b/c there are some extra commits and liferay-frontend hasn't been updated yet

**Comment by john-co** [comment]
Closing since I can't fully update the commits with my edits.

**Pull request closed without merging**